packages/e2e-test-utils-playwright/src/editor/site-editor.ts

```diff
@@ -7,8 +7,12 @@ import type { Editor } from './index';
  * Save entities in the site editor. Assumes the editor is in a dirty state.
  *
  * @param this
+ * @param isOnlyCurrentEntityDirty If the only dirty entity is the current one, skip trying to open the save panel.
  */
-export async function saveSiteEditorEntities( this: Editor ) {
+export async function saveSiteEditorEntities(
+	this: Editor,
+	isOnlyCurrentEntityDirty: boolean
```

Comment:
Let's make this an object param. The `boolean` arguments don't scale well, IMO.

cc @kevin940726, @WunderBart

Reply:
I can do that, yes. Although I wouldn't expect any more params here, but you never know 😄